### PR TITLE
[Shipping] Remove class Claculator, all calculators implement CalculatorInterface

### PR DESCRIPTION
**Pull request by tuka217**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | yes
| Deprecations? | no
| Fixed tickets | -
| License       | MIT
| Doc PR        | -

### CODE REVIEW

File: src/Sylius/Component/Shipping/CHANGELOG.md [hunk 1]
```diff
@@ -4,3 +4,8 @@ CHANGELOG
 ### v0.10.0
 
 * Initial dev release.
+
+### 19-11-2015
+
+* Remove `Calculator`
+* All calculators class implements `CalculatorInterface`
```
Comment (pjedrzejewski) [inline src/Sylius/Component/Shipping/CHANGELOG.md]: I know we miss previous versions, but let's add this to v0.16.0 section. Kudos for remembering about changelog. ;)

**Comment by pjedrzejewski** [comment]
:+1:

**Comment by pjedrzejewski** [comment]
Nice work Ania! Thanks! :)

**Comment by michalmarcinkowski** [comment]
@pjedrzejewski @tuka217 what about updating CHANGELOG.md file?

**Pull request merged**